meta-schema.graphql

```diff
@@ -0,0 +1,94 @@
+# Base types
+type Identifiers {
+	id:ID!
+	idType:String! # IPI || ISWC || GRid
+}
+
+type Party {
```

Comment:
Should we have metaId field as part of Party ?

Reply:
That would just be an `idType`.

Reply:
You mean another identifier (type METAID) .

Reply:
Yes